_posts/2024-04-05-spidermonkey.md

```diff
@@ -0,0 +1,98 @@
+
+Other examples can be found in the related issues that have been closed:
+- [WedIDL: bring dom/bindings/typedarray further in line with spec](https://github.com/servo/servo/issues/31319)
+- [WebIDL: use TypedArray](https://github.com/servo/servo/issues/31064)
+- [Remove create_typed_array from dom/bindings](https://github.com/servo/servo/issues/31050)
+- [WebIDL: use FLoat32Array in GamePad](https://github.com/servo/servo/issues/31049)
+- [WebIDL: use FLoat32Array in XRRay](https://github.com/servo/servo/issues/31048)
+- [WebIDL: use FLoat32Array in XRRigidTransform](https://github.com/servo/servo/issues/31047)
+- [WebIDL: use FLoat32Array in XRView](https://github.com/servo/servo/issues/31046)
+- [WebIDL impl: remove unsafe JSObject from return value of Document::NamedGetter](https://github.com/servo/servo/issues/30890)
+
+As this work continues, an increasing amount of SpiderMonkey specifics will be moved to [`components/script/dom/bindings`](https://github.com/servo/servo/tree/58081579e9a537ba6bd71bcdcb2b066e14e037b8/components/script/dom/bindings), which will make it possible to enumerate what we are doing with our script engine, and to start investigating how this could be expressed through a generic interface: one that any bindings layer to a script execution engine could implement to integrate with Servo.
+
+### Open issues
+
+- [Modular JS/execution engine](https://github.com/servo/servo/issues/30863)
+- [WebIDL impl: Replace use of NonNull<JSObject>](https://github.com/servo/servo/issues/30889)
+- [WebIDL impl: remove unsafe JSObject when returning a ReadableStream](https://github.com/servo/servo/issues/30891)
+- [WebIDL impl: remove unsafe JSObject from WebGLExtensionWrapper](https://github.com/servo/servo/issues/30892)
+- [Support FinalizationRegistry](https://github.com/servo/servo/issues/31072)
+
+--------
+
+Thank you for reading. This report was written by [Gregory Terzian](https://github.com/gterzian). Questions and comments can be made by opening an issue at [the original repo](https://github.com/gterzian/spidermonkey_servo), or by [reaching out on Zulip](https://servo.zulipchat.com). 
```

Comment:
About authorship, blog posts are from Servo as a whole, so won't say anything on the blog post itself. On the report, you can write the author and date (similar to what we have done in the last report: https://github.com/servo/servo/wiki/Servo-Layout-Engines-Report).

I think for questions, it'd be better to fill issues on Servo's GitHub directly.

Reply:
Ok, thanks for the info, have added the report to https://github.com/servo/servo/wiki/Servo-and-SpiderMonkey-Report, and updated the link at https://github.com/servo/servo/wiki/Design#script

Will turn this into a short blog post.

Reply:
Cool, thanks!